identifiers/vendor.txt

```diff
@@ -185,10 +199,13 @@ Datamax
 DeTeWe - Deutsche Telephonwerke AG
 Debian
 Dell
+Dell EMC
```

Comment:
I don't see the corresponding `Dell EMC` vendor in any of the fingerprints.  When I do complex / larger PRs I often delete all of the identifier files and regenerate them from data using `ruby bin/recog_standardize`.  I can then look at the diff to sanity check them and it removes unused values.

Reply:
Ah, interesting. I had deleted+regenerated the other other day, and it's still there! Looks like it appears in `xml/mdns_replies.xml` (I'm open to feedback if it'd be better to make that entry just `Dell` or `EMC`).

Reply:
Good catch, will fix here. :+1: